core/trino-main/src/test/java/io/trino/sql/query/TestColumnMask.java

```diff
@@ -218,6 +218,96 @@ public void testMultipleMasksOnDifferentColumns()
                 .matches("VALUES (BIGINT '-370', 'X')");
     }
 
+    @Test
+    public void testMultipleMasksUsingOtherMaskedColumns()
+    {
+        String query = "SELECT comment, orderstatus, clerk FROM orders WHERE orderkey = 1";
+        String expected = "VALUES (CAST('nstructions sleep furiously among ' as varchar(79)), 'O', 'Clerk#000000951')";
+
```

Comment:
it would be worth for the readability of the test scenarip to perform the query without any column masking